python/python-hello-world/README.md

```diff
@@ -7,129 +7,48 @@
-  - Cloud Code usses the .vscode/launch.json file to locate the pods to attach a debug session to
-  - the hello-world container was built to run `ptvsd` to support debugging
-- You should see the [Debug toolbar](https://code.visualstudio.com/Docs/editor/debugging#_debug-actions) with familiar debug controls
-- You can click in the margin in `app.py` to add a breakpoint. When you send a new request to your application, it will pause at the desired line
-- You can inspect variables and stack info in the [Debug Side Bar](https://code.visualstudio.com/Docs/editor/debugging)
-- You can interact with the debugging session using the [Debug Console](https://code.visualstudio.com/Docs/editor/debugging#_debug-console-repl) in the bottom pane
-![debugger](./img/debug.gif)
-
-### Open a Terminal in Your Container
-By opening a terminal, you can monitor and debug running processes as though they were running on your local machine
-- Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
-- Type "CloudCode: Get Terminal"
-- Select the pod you wish to connect to
-- You now have access to a shell inside the running container
-![container terminal](./img/term.gif)
-
-### Tasks
-This sample also includes a number of [Visual Studio Code Tasks](https://code.visualstudio.com/docs/editor/tasks) to launch common commands
-
-#### Run the Pylint Task
-Run a Python [linter](https://en.wikipedia.org/wiki/Lint_(software)) to analyze your code for issues
-- [Install Pylint](https://www.pylint.org/#install) if necessary
-- Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
-- Type "Tasks: Run Task"
-- Select the "Pylint" task
-- An output window will appear showing the Pylint output
-![lint task](./img/lint.gif)
+This sample was written to demonstrate how to use the Cloud Code extension for Visual Studio code.
+* [Install Cloud Code for VS Code](https://cloud.google.com/code/docs/vscode/install)
+* [Creating a new app](https://cloud.google.com/code/docs/vscode/creating-an-application)
+* [Editing YAML files](https://cloud.google.com/code/docs/vscode/yaml-editing)
+
 
 ---
 
 ## Using the Command Line
 As an alternative to using the Cloud Code extension, the application can be deployed to a cluster using standard command line tools
 
-### Using Skaffold
+### Skaffold
 [Skaffold](https://github.com/GoogleContainerTools/skaffold) is a command line tool that can be used to build, push, and deploy your container images as you work
 
 ```
 skaffold run --default-repo=gcr.io/your-project-id-here/cloudcode
 ```
 
-### Using kubectl
+### kubectl
 
 [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is the official Kubernetes command line tool. It can be used to deploy Kubernetes manifests to your cluster, but images must be build seperately using another tool (for example, using the [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/)) 
 
```

Comment:
(sorry couldn't comment outside the diff but) 

delete "Build container images" section below, doesn't make any sense given there's already skaffold.

similarly, delete the "Deploy manifests" section too.

Reply:
SG